### PR TITLE
Add application-profile icon

### PR DESCRIPTION
**Pull request by ketsappi**
## Description

Adds new icon, application-profile.

## Release notes

- Add `application-profile` icon

### CODE REVIEW

File: src/baseIcons/application-profile.svg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>Icons/Basic icons/application-profile</title>
+    <g id="Icons/Basic-icons/application-profile" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <path d="M8,15 C8,15.553 7.552,16 7,16 L1,16 C0.448,16 0,15.553 0,15 L0,9 C0,8.447 0.448,8 1,8 L7,8 C7.552,8 8,8.447 8,9 L8,11 L11,11 L11,8 L9,8 C8.448,8 8,7.553 8,7 L8,1 C8,0.447 8.448,0 9,0 L15,0 C15.552,0 16,0.447 16,1 L16,7 C16,7.553 15.552,8 15,8 L13,8 L13,11 L20,11 L20,11 C20.552,11 21,11.447 21,12 L21,16.001 L23,16.001 C23.552,16.001 24,16.448 24,17.001 L24,23 C24,23.553 23.552,24 23,24 L17,24 C16.448,24 16,23.553 16,23 L16,17.001 C16,16.448 16.448,16.001 17,16.001 L19,16.001 L19,13 L8,13 L8,15 Z M22,18.001 L18,18.001 L18,22 L22,22 L22,18.001 Z M6,10 L2,10 L2,14 L6,14 L6,10 Z M14,2 L10,2 L10,6 L14,6 L14,2 Z" id="Combined-Shape" fill="#222222"></path>
+    </g>
+</svg>
```
Comment (riitasointi) [inline src/baseIcons/application-profile.svg]: I think this file should follow the established naming convention: `icon-XXXXX.svg`

**Review by riitasointi — CHANGES_REQUESTED** [review]

File: src/baseIcons/application-profile.svg (hunk 1 above)
Comment (ketsappi) [reply src/baseIcons/application-profile.svg]: That is totally true. Good catch! Will fix this.

Comment (ketsappi) [reply src/baseIcons/application-profile.svg]: This fix is now done and available in this commit https://github.com/vrk-kpa/suomifi-icons/pull/69/commits/a1ad325031432da3d1f6f2db5d5ff83e1b0220bb

**Review by riitasointi — APPROVED** [review]
Looks good!

**Pull request merged**